responsibleai_vision/responsibleai_vision/managers/explainer_manager.py

```diff
@@ -598,9 +597,9 @@ def _get_fail_str(self):
         draw.text((x, y), text, fill="white", font=font)
         fail.show()
         fail.save('fail.jpg')
-        image = get_image_from_path("fail.jpg", "RGB")
-        jpg_img = cv2.imencode('.jpg', image)
-        return base64.b64encode(jpg_img[1]).decode('utf-8')
+        byte_buffer = io.BytesIO()
```

Comment:
what was the reasoning behind making this change here?  not sure what the issue was that required this fix.